sphinx_doc_src/cmds/string.rst

```diff
@@ -48,6 +50,17 @@ Most subcommands accept a ``-q`` or ``--quiet`` switch, which suppresses the usu
 
 The following subcommands are available.
 
+"collect" subcommand
+--------------------
+
+``string collect [(-c | --chomp)] [STRING...]``
+
+``string collect`` collects its input into a single output argument, without splitting the output when used in a command substitution. This is useful when trying to collect multiline output from another command into a variable.
+
+If invoked with multiple arguments instead of input, ``string collect`` preserves each argument separately, where the number of output arguments is equal to the number of arguments given to ``string collect``.
+
+``--chomp`` trims a single trailing newline off of each output argument. This is useful when collecting the output from another command as the trailing newline is frequently not desired.
```

Comment:
I'm not entirely sold on calling this "chomp". It's a perl-ism, so some people might know it, but I don't think it's intuitive.

How about being boring and calling it "trim-newline" instead?

Reply:
I've always considered "chomp" a generic term for "trim a single trailing newline". I dabbled slightly with perl around 2 decades ago but haven't used it since then. Is "chomp" really that perl-specific?

Reply:
Done. I picked `-n` as the short flag.